tests/acceptance/features/apiSpaces/changingFilesShare.feature

```diff
@@ -0,0 +1,40 @@
+@api @skipOnOcV10
+Feature:
+
+  Background:
+    Given using spaces DAV path
+    And these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+
+
+  @issue-4421
+  Scenario: Move files between shares by different users
+    Given user "Carol" has been created with default attributes and without skeleton files
+    And user "Alice" has uploaded file with content "some data" to "/textfile0.txt"
+    And user "Alice" has created folder "/PARENT"
+    And user "Brian" has created folder "/PARENT"
+    And user "Alice" has moved file "textfile0.txt" to "PARENT/shared_file.txt" in space "Personal"
+    And user "Alice" has shared folder "/PARENT" with user "Carol"
+    And user "Brian" has shared folder "/PARENT" with user "Carol"
+    And user "Carol" has accepted share "/PARENT" offered by user "Alice"
+    And user "Carol" has accepted share "/PARENT" offered by user "Brian"
+    When user "Carol" moves file "PARENT/shared_file.txt" to "PARENT (1)/shared_file.txt" in space "Shares Jail" using the WebDAV API
+    Then the HTTP status code should be "502"
+    And for user "Carol" folder "PARENT" of the space "Shares Jail" should contain these entries:
+      | shared_file.txt |
+    And for user "Carol" folder "PARENT (1)" of the space "Shares Jail" should not contain these entries:
+      | shared_file.txt |
+
+
+  Scenario: overwrite a received file share
+    Given user "Alice" has uploaded file with content "this is the old content" to "/textfile1.txt"
```

Comment:
Maybe this can also go inside `shareSpace.feature`?

Reply:
IMO, the scenario is more related about changing the received share rather moving between the spaces